boards/arm64/qemu_cortex_a53/qemu_cortex_a53_smp_defconfig

```diff
@@ -25,3 +25,4 @@ CONFIG_MP_NUM_CPUS=2
 CONFIG_CACHE_MANAGEMENT=y
 CONFIG_TIMEOUT_64BIT=y
 CONFIG_ARMV8_A_NS=y
+CONFIG_MAX_XLAT_TABLES=12
```

Comment:
It's a bit odd that you need to increase the number of xlat tables. I suggest you to investigate why you actually need this, I suspect this is hiding a different problem somewhere.

Reply:
Yes, it is odd, I can't find out why my pathces impact qemu_cotex_a53_smp board. most likely I modify GICV2 driver, but it seems qemu_cotex_a53_smp board is using GICv3, right?

Reply:
I think the problem is that setting the VA/PA size to 40-bits there are now more tables to deal with when copying / duplicating, if this is correct then it's fine to increment the number of XLAT tables here. I'll try to investigate, meanwhile feel free to increment this number to the minimum needed to pass the tests.

Reply:
I removed SMP related patches but keep VA/PA 40bit patch, now all checks have passed, so the issue should be not introduced by VA/PA 40bit patches, I would like to submit non-SMP patches firstly, and then submit SMP patches in another pull request, thanks.